asttoc/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.5)
+
+set(LLVM_ENABLE_RTTI ON)
```

Comment:
How come you're enabling RTTI here? I'm building LLVM with RTTI off as that's what OSL uses.

Reply:
It's fine to remove this now. I originally planned on using cereal to read the json and cereal needed typeid, but I'm not using cereal anymore.

Reply:
FYI, current OSL no longer disables RTTI across the board, that was causing problems with other dependencies that turned out to *require* RTTI. Now we only disable it [for the one file that directly calls LLVM API calls](https://github.com/AcademySoftwareFoundation/OpenShadingLanguage/blob/master/src/liboslexec/CMakeLists.txt#L206). I isolate the LLVM calls in one wrapper class module, which doesn't need to call into any other dependencies that might use RTTI. You can make the RTTI on/off choice module-by-module, as it turns out.

Reply:
My memory might be hazy. Now that I think about it, I believe it was not OSL's dependencies that were the problems, but downstream apps that needed to link against both OSL (which was building RTTI off across the board) and something else that required RTTI, and was having trouble with that mix. So I had to stop "exporting" the no-RTTI requirement from OSL, and instead restrict it to just the one module that directly called LLVM which is usually built without RTTI.

Reply:
Ah that's good to know, thanks Larry.